tests/jerry/es2015/object-get_own_property_symbols.js

```diff
@@ -0,0 +1,84 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Test array
+var a = Symbol ('a')
+var b = Symbol ('b')
+var c = Symbol ('c')
+var d = Symbol ()
+
+var arr = [a, b, c];
+var props = Object.getOwnPropertySymbols(arr);
+assert (props.indexOf("0") === -1);
+assert (props.indexOf("1") === -1);
+assert (props.indexOf("2") === -1);
+assert (props.indexOf("length") === -1);
+assert (props.length == 0);
+
+// Test object
+var obj = {};
+obj[a] = 'a';
+obj[b] = 'b';
+obj[c] = 'c';
+obj[d] = 'd';
+props = Object.getOwnPropertySymbols(obj);
+// props should not contain: a, b, c, d and the order is not defined!
+assert (props.indexOf(a) !== -1);
+assert (props.indexOf(b) !== -1);
+assert (props.indexOf(c) !== -1);
+assert (props.indexOf(d) !== -1);
+assert (props.length === 4);
```

Comment:
If assertion shall be made on properties keyed with same description symbols? Since symbols with same descriptions shall not be treated as same key.

Reply:
Well, that's a good idea! I'll add a new test case for it.